static/menu.js

```diff
@@ -0,0 +1,18 @@
+const navigationBar = $("#navbar");
+
+$("#contentBox").css("top", $("#navbar").css("height")); //Sets the content below the navbar.
+
+document.onscroll = () => {
```

Comment:
This uses ES6 which I think is fine, but that means we need an asset pipeline to transpile it since we will have people using IE.